workers/api-server/src/http/admin.rs

```diff
@@ -272,9 +272,10 @@ impl TypedHandler for AdminWalletOrderIdsHandler {
         let wallet_id = parse_wallet_id_from_params(&params)?;
         let wallet =
             self.state.get_wallet(&wallet_id).await?.ok_or(not_found(ERR_WALLET_NOT_FOUND))?;
-        let order_ids = wallet.orders.keys().cloned().collect();
 
-        Ok(AdminWalletOrderIdsResponse { order_ids })
+        let order_ids = wallet.get_matchable_orders().into_iter().map(|o| o.0).collect();
```

Comment:
nit: for readability can we change the map closure to `|id, _order| id`? It's unclear just by reading it what types we're working with